tools.go

```diff
@@ -1,3 +1,4 @@
+//go:build tools
```

Comment:
Can you please elaborate why to make this change?

Reply:
This comes from go 1.17.6 & `make code/fix`(_goimports_). 

[More info](https://go.dev/doc/go1.17#go-command): -> Search for "_//go:build lines_"

According to the docs, this does not seem to introduce any breaking changes with older versions.